### PR TITLE
Adjust Width of Creator Settings Form Fields 

### PR DESCRIPTION
**Pull request by juliannatetreault**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
This PR adjusts the width of the `community_name` and `primary_brand_color_hex` fields and of the "Finish" button on the Creator Settings form to be a max of 480px per [this](https://github.com/forem/forem/pull/15725#issuecomment-992085874) PR comment.

## Related Tickets & Documents
Relates to PR #15725 

#### Please note that you will need to essentially clear out your database to take this feature for a spin. 🏎️  

To do so, you can either drop your entire database via `rails:db drop` 

_**or**_

You can open a rails console via `rails c` and do the following:
 ```
# Enable the feature flag
FeatureFlag.enable(:creator_onboarding)

# Delete all users 
User.destroy_all

# Set `waiting_on_first_user` to be true
Settings::General.waiting_on_first_user = true

# If you've logged in before set the mascot id to nil
 Settings::General.mascot_user_id = nil
```

Now, run your server and navigate to `http://localhost:3000/` and complete the "Create Account" form. After completing the "Create Account" form, you should be brought to the new Creator Setup page, `${baseUrl}/admin/creator_settings/new?referrer=${baseUrl}`.

Once on the Creator Settings page, you will want to test the changes outlined below:
#### Before:
Screen Shot 2021-12-10 at 6 11 22 AM

#### After:
Screen Shot 2021-12-13 at 1 17 20 PM

#### After (Mobile View):
Screen Shot 2021-12-13 at 1 17 37 PM

### UI accessibility concerns?
N/A 🤞 

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: the Creator Settings is already covered by tests.
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

Futurama: Fry saying "Fix it!"

### CODE REVIEW

File: app/assets/stylesheets/setup-mode.scss [hunk 1]
```diff
@@ -6,6 +6,10 @@ body.default-header {
   padding-top: 0;
   min-height: 100vh;
 }
+.creator-settings-fields {
+  height: 40px;
+  max-width: 480px;
+}
```
Comment (juliannatetreault) [inline app/assets/stylesheets/setup-mode.scss:12]: I took the height from the Figma design and added the `max-width` as `480px` per this comment. However, I wonder if there's a better way to do this via utility classes/Crayons.

**Comment by juliannatetreault** [comment]
I feel like the text looks a little bit weird with the adjusted width of the `community_name` and `primary_brand_color_hex` fields and the adjusted width of the "Finish" button since it runs past the fields when not in mobile view. 

cc @anujbhavsar96

**Comment by ludwiczakpawel** [comment]
Sorry I've just realized I requested changed even though it was a draft... :x sorry about that!

**Comment by juliannatetreault** [comment]
> Hey, I think all you have to do is to change `crayons-layout--limited-s` to be `crayons-layout--limited-xs` here:
> 
> https://github.com/forem/forem/blob/main/app/views/admin/creator_settings/new.html.erb#L6
>
> This will automatically set 480px max width for the entire layout. This is also how it's done on `/app/views/shared/authentication/_forem_creator_signup.html.erb` so I'm actually wondering if there was a reason to change it to be `s` instead of `xs`...?

Ah, thank you for this, @ludwiczakpawel! I knew there must be a better way. 😅 

> Anyway, this is one of the nuances we have for Crayons layouts that are obviously not documented anywhere _sigh_ but I'm working on that...

Don't worry about it! We'll get there with time. 😊

**Review by ludwiczakpawel — APPROVED** [review]

**Review by nickytonline — APPROVED** [review]
Let's go! 🚀

**Review by citizen428 — APPROVED** [review]

**Review by Ridhwana — APPROVED** [review]

**Pull request merged**